### DISCUSSION TITLE
Free Tier Details / Launch Date

### DISCUSSION THREAD

**Discussion by trymbill** [nhost/nhost#81] [Q&A] [answered]
`nhost` looks exactly like what I've been searching for, love the idea and implementation.

Regarding the free tier that's both talked about on the website and in this repo's README. Are there any more details available around the logistics of that tier, how it will work and what the "premium" features will be? I tried searching for more information but came up short. Also, is there a launch date planned for the free tier?

**Comment by sebagudelo** [comment]
Hey @trymbill our apologies for the slow reply.

Nhost 2.0 will provide a new developer experience. It will be encompass all of the learnings from the last 24 months building Nhost. We will have a new approach to Domains, Storage, CLI, Functions and Console.

The free tier will be best suited to new projects just starting up.

It will have all aspects of the new developer experience with certain limits on Database, Network Traffic, Storage and Team Based Functionality.

We will be soft launching over the next couple of weeks. 

You can sign up below to be notified:
https://waitlist.nhost.io/

If you have an urgent project requirement, please email: seb@nhost.io and we can discuss further.

**Comment by elitan** [comment, marked as answer]
The free tier is launched 🚀 🚀 🚀 

Pricing and features available here: https://www.nhost.io/pricing

Sign up and start building: https://app.nhost.io/
